main/guides/orchestration/contract-walkthroughs/send-anywhere.md

```diff
@@ -81,7 +81,7 @@ these functions with the necessary context (such as the contract state, logging,
 const orchFns = orchestrateAll(flows, {
   log,
   sharedLocalAccountP,
-  zoeTools,
+  zoeTools
```

Comment:
Lint insistes on `no-trailing-comma`, which is incorrect for our JavaScript style. Is it worth finding and fixing this?

Reply:
not worth fixing imo